frontend/src/scenes/web-analytics/WebPropertyFilters.tsx

```diff
@@ -11,11 +11,43 @@ import { useState } from 'react'
 import { webAnalyticsLogic } from './webAnalyticsLogic'
 
 export const WebPropertyFilters = (): JSX.Element => {
-    const { rawWebAnalyticsFilters } = useValues(webAnalyticsLogic)
+    const { rawWebAnalyticsFilters, preAggregatedEnabled } = useValues(webAnalyticsLogic)
     const { setWebAnalyticsFilters } = useActions(webAnalyticsLogic)
 
     const [displayFilters, setDisplayFilters] = useState(false)
 
+    const taxonomicGroupTypes = [
+        TaxonomicFilterGroupType.EventProperties,
+        TaxonomicFilterGroupType.SessionProperties,
+        ...(!preAggregatedEnabled ? [TaxonomicFilterGroupType.PersonProperties] : []),
```

Comment:
What's the idea here, in the future we'll let customers choose whether they wanna use this or not, and hide some filters in that case?

Reply:
We want to ask people to try those while in beta and we can't support person properties on pre-aggregated tables, so it is easier to show only what it is supported and increment the list when we add more stuff.

In the future, if we make the pre-aggregated tables the default for everyone, we can just de-optimize the query if those filters are included (which already happens)

We are prioritizing the filters available based on usage as each new one includes an overhead to the group by/merges: https://us.posthog.com/project/2/notebooks/tM6DMvnQ